### PR TITLE
fix stuff + functions

### PR DESCRIPTION
**Pull request by absoIute**
👍

### CODE REVIEW

File: bindings/2.206/GeometryDash.bro [hunk 1]
```diff
@@ -5593,7 +5592,7 @@ class GameOptionsLayer : GJOptionsLayer {
 	static GameOptionsLayer* create(GJBaseGameLayer*);
 
 	bool init(GJBaseGameLayer*);
-	void onPracticeMusicSync(cocos2d::CCObject* sender);
+	void onPracticeMusicSync(cocos2d::CCObject* sender) = 0x28e010;
```
Comment (dankmeme01) [inline bindings/2.206/GeometryDash.bro:5595]: where's the win

**Comment by absoIute** [comment]
what the flip

File: bindings/2.206/GeometryDash.bro [hunk 2]
```diff
@@ -5904,7 +5903,7 @@ class GauntletLayer : cocos2d::CCLayer, LevelManagerDelegate {
 
 	static GauntletLayer* create(GauntletType);
 
-	bool init(GauntletType);
+	bool init(GauntletType) = 0x1e93d0;
```
Comment (Prevter) [inline bindings/2.206/GeometryDash.bro:5906]: another oopsie

**Review by Prevter — CHANGES_REQUESTED** [review]

**Review by Prevter — APPROVED** [review]

File: bindings/2.206/GeometryDash.bro [hunk 3]
```diff
@@ -4018,7 +4018,6 @@ class FMODAudioEngine : cocos2d::CCNode {
 	PAD = win 0x8, android32 0x8, android64 0x8;
 	float m_pulse1;
 	float m_pulse2;
-	TodoReturn canItemBeUnlocked(int, UnlockType);
```
[the remark this answers is not in the document]
Comment (absoIute) [reply bindings/2.206/GeometryDash.bro:4021]: the function isn't part of that class and its incorrectly amongst member variables

**Pull request merged**